Marlin/src/HAL/HAL_STM32F1/HAL_sdio_STM32F1.cpp

```diff
@@ -21,7 +21,7 @@
  *
  */
 
-#ifdef __STM32F1__
+#if defined(__STM32F1__) && (defined(STM32_HIGH_DENSITY) || defined(STM32_XL_DENSITY))
```

Comment:
Should this also check for `STM32F1xx`?

Reply:
There is no need for STM32F1xx in this check.
STM32F1 defines are a bit messy.
There are __STM32F1__, STM32F1 and STM32F1xx
STM32F1 and STM32F1xx are used in pair and __STM32F1__ is used alone.